example/tsconfig.json

```diff
@@ -10,7 +10,9 @@
     "experimentalDecorators": true,
     "strict": true,
     "suppressImplicitAnyIndexErrors": true,
-    "removeComments": true
+    "removeComments": true,
+    "jsx": "preserve",
+    "noImplicitAny": false
```

Comment:
OK, setting `noImplicitAny` is a workaround. But I would still recommend to add JSXElement declaration.